packages/nr-metadata/nr_metadata/ui/semantic-ui/js/nr/forms/components/CreatibutorsField/CreatibutorsModal.jsx

```diff
@@ -604,13 +604,23 @@ export const CreatibutorsModal = ({
                 <Button
                   name="submit"
                   type="submit"
-                  onClick={() => {
+                  onMouseDown={() => {
                     setAction("saveAndContinue");
                     setShowPersonForm(
                       autocompleteNames !== NamesAutocompleteOptions.SEARCH_ONLY
                     );
                     handleSubmit();
                   }}
+                  onKeyDown={(e) => {
+                    if (e.key === "Enter") {
```

Comment:
Small suggestion - i would extract insides to `handleSaveAndContinue` function, same for `saveAndClose`  to reduce duplication